app/src/main/java/com/client/fire_and_water/FriendsFragment.kt

```diff
@@ -0,0 +1,78 @@
+package com.client.fire_and_water
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import com.client.fire_and_water.databinding.FragmentFriendsBinding
+
+class FriendsFragment : Fragment() {
+
+    lateinit var listAdapter: ArrayAdapter<String>
+    lateinit var friendNames: ArrayList<String>
+    private var _binding: FragmentFriendsBinding? = null
+    private val binding get() = _binding!!
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentFriendsBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        friendNames = getDummyFriendsList()
```

Comment:
Isn't it easier then to initialize list here? Or is it planned that the list will be updated?